src/Lykke.Blockchains.Integrations.Contract/Common/Cryptograhy/AesKeysEnvelope.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Lykke.Blockchains.Integrations.Contract.Common.Cryptograhy
```

Comment:
Typo in namespace: not Cryptograhy but Cryptography

Reply:
fixed